testing-modules/junit-5/src/main/java/groupedvsmultipleassertions/User.java

```diff
@@ -0,0 +1,37 @@
+package groupedvsmultipleassertions;
```

Comment:
Please can we have this in the `com.baeldung.groupedvsmultipleassertions` package?